Cargo.toml

```diff
@@ -20,17 +20,16 @@ futures = "0.3.5"
 log = "0.4.11"
 metrics = { version = "0.12.1", features = ["std"] }
 rand = { version = "0.7", default-features = false, features = ["small_rng"] }
-rusoto_cloudwatch = "0.45"
-rusoto_core = "0.45"
+rusoto_cloudwatch = "0.46"
+rusoto_core = "0.46"
 serde_json = "1"
-stream-cancel = "0.6.1"
-tokio = { version = "0.2.21", features = ["full"] }
+tokio = { version = "1", features = ["time"] }
```

Comment:
Isn't `tokio::sync::mpsc` supposed to only be available with the `sync` feature?

Reply:
Yes, but hyper already enables it which is why it works here.